src/test/java/com/EighthLight/app/TestBoard.java

```diff
@@ -0,0 +1,83 @@
+package com.EighthLight.app;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+class TestBoard {
+
+    private Board testBoard;
+    private int size;
+
+    @BeforeEach
+    public void setUp() {
+        size = 3;
+        testBoard = new Board(size);
+        testBoard.createBoard();
+    }
+
+    @Test
+    void createBoardMakesAnArrayFilledWithInts() {
+        ArrayList<Integer> expectedBoard = new ArrayList<Integer>(size * size);
+        for (int i = 0; i < size*size; i++) {
+            expectedBoard.add(i);
+        }
+
+        assertEquals(expectedBoard, testBoard.getSpaces());
+    }
+
+    @Test
+    void spaceWithinBoundsReturnsTrueIfSpaceIsWithinBounds() {
+        String userInput = "0";
+
+        assertTrue(testBoard.spaceWithinBounds(userInput));
+    }
+
+    @Test
+    void spaceWithinBoundsReturnsFalseIfSpaceIsNotWithinBounds() {
+        String userInput = "13";
+
+        assertFalse(testBoard.spaceWithinBounds(userInput));
+    }
+
+    @Test
+    void spaceWithinBoundsReturnsFalseIfInputIsNotANumber() {
+        String userInput = "S";
+
+        assertFalse(testBoard.spaceWithinBounds(userInput));
+    }
+
+    @Test
+    void spaceWithinBoundsReturnsFalseIfInputIsEmpty() {
+        String userInput = "";
+
+        assertFalse(testBoard.spaceWithinBounds(userInput));
+    }
+
+    @Test
+    void updateSpaceUpdatesSpaceWithUserInput() {
+        String userInput = "1";
+        String userSymbol = "X";
+        ArrayList expectedBoard = new ArrayList(Arrays.asList(0, "X", 2, 3, 4, 5, 6, 7, 8));
+        MockPlayer playerOne = new MockPlayer("X");
+        testBoard.updateSpace(userInput, playerOne);
```

Comment:
why does the board need to know about the entire player? Wouldn't it be enough for it to only know about the Player's symbol?